lib/engines/content_object_store/app/components/content_object_store/content_block/document/show/summary_list_component.rb

```diff
@@ -57,6 +59,20 @@ def creator_item
     }
   end
 
+  def state_item
+    {
+      field: "State",
+      value: content_block_document.latest_edition.state,
+    }
+  end
+
+  def scheduled_item
+    {
+      field: "Scheduled for publication at",
+      value: content_block_document.latest_edition.scheduled_publication&.strftime("%d %B %Y at %I:%M%P"),
```

Comment:
We should use `%e` here to show the date without as leading zero, as per the [GOV.UK guidance on dates](https://www.gov.uk/guidance/style-guide/a-to-z#dates), i.e:

```suggestion
      value: content_block_document.latest_edition.scheduled_publication&.strftime("%e %B %Y at %I:%M%P"),
```

Reply:
good spot - I will do this in a separate commit as we are using this format elsewhere so will update everywhere (maybe needs our own entry in `format` as it looks like Whitehall don't have the same rendering? I got this from the figma designs and think it is more up to date in terms of guidelines...)

Reply:
It might be worth adding the format to en.yml along with the other date formats (https://github.com/alphagov/whitehall/blob/3abce9256c2ca443ec7b1189d4a6c166ec3f2c2d/config/locales/en.yml#L479) and then using i18n to format the time ie:

```ruby
value: l(content_block_document.latest_edition.scheduled_publication, format: :human_readable)
```